sc_rf_service/sc_rf_service.py

```diff
@@ -91,45 +107,68 @@ class CavityPV(PVGroup):
     ssaOnOff_Act = pvproperty(value=1, name=':IS_ON', dtype=ChannelType.ENUM,
                               enum_strings=("F", "T"))
     ssaLatch = pvproperty(value=0, name=":SSA_LTCH", dtype=ChannelType.ENUM,
-                          enum_strings=("OK", "Faulted"))
+                          enum_strings=("OK", "Fault"))
     ssaAlarmSum = pvproperty(value=0, name=":SSA:AlarmSummary.SEVR",
                              dtype=ChannelType.ENUM,
-                             enum_strings=("", "", "Faulted", "Faulted"))
+                             enum_strings=("NO_ALARM", "MINOR", "MAJOR", "INVALID"))
```

Comment:
Thank you for actually looking at these, I was too lazy when I originally added it hahaha